internal/core/application/operator_service.go

```diff
@@ -814,6 +827,294 @@ func (o *operatorService) FeeAccountBalance(ctx context.Context) (
 	return int64(baseAssetAmount), nil
 }
 
+type depositType string
+
+var (
+	marketDeposit depositType = "MarketDeposit"
+	feeDeposit    depositType = "FeeDeposit"
+)
+
+// ClaimMarketDeposit method add unspents to the market
+func (o *operatorService) ClaimMarketDeposit(
+	ctx context.Context,
+	marketReq Market,
+	outpoints []TxOutpoint,
```

Comment:
if we used the domain.OutpointWithAsset type here, the marketReq argument would become useless.

I think this would make the life of the operator a bit easier.. he would be only concerned with adding reference to some utxos and their asset without adding a market or leaving it empty.

Reply:
I do believe the opposite, is better to have the operator to be explicit in what market is going to claim and prepares him to all future workflows will eventually require already him to specify the market always., it works as self-documenting and self-confirming action.  I need this for operator frontends, gives a better clear framework for making dashabord etc..